src/components/MapView/utils.ts

```diff
@@ -12,6 +12,28 @@ function getBounds(waypoints: Array<[number, number]>, directionCoordinates: und
     };
 }
 
+function haversineDistance(coordinate1: number[], coordinate2: number[]) {
```

Comment:
Please add JS docs to this function so everyone knows what this is doing, I had to google what harvesine is haha